docs/formatters-spec.md

```diff
@@ -1,45 +1,36 @@
-# Formatter spec
+# Formatter specification
 
-In order to keep the design of treefmt simple, we want all of the formatters
-to adhere to a unified specification. This document outlines that spec.
+In order to keep the design of `treefmt` simple, we support only formatters which adhere to a certain standard. This document outlines this standard. If the formatter you would like to use doesn't comply with the rules, you should create a wrapper script that transforms the usage to match the specification.
 
 ## Command-line interface
 
-A formatter MUST adhere to the following interface:
```

Comment:
This document's style with "MUST" and "SHOULD" was meant to match https://www.rfc-editor.org/rfc/rfc2119

As a specification, you want the style to be even more declarative, almost like programming.